openfisca_uk/data/datasets/frs/enhanced/stages/calibration/losses/loss_category.py

```diff
@@ -21,7 +21,7 @@ def weighted_squared_relative_deviation(
     Returns:
         tf.Tensor: The weighted squared relative deviation.
     """
-    return (pred - actual) ** 2
+    return (pred / (actual + 1) - 1) ** 2
```

Comment:
Is this because there are actuals of zero? What's an example if so?

Reply:
Yes, legacy benefits which are phased out. Also the I Council Tax band for some regions.